docs/tutorial/using-checkbox/basic-launcher.rst

```diff
@@ -0,0 +1,260 @@
+you want to upload the test results to the Canonical :term:`Certification
+website`. Let's say you don't need to upload the results there; you are only
+interested in the text summary and the test reports.
+
+Edit the launcher file:
+
+.. code-block:: none
+    :caption: mylauncher
+    :name: launcher-stock-reports
+    :emphasize-lines: 4
+
+    [launcher]
+    launcher_version = 1
+    app_id = com.canonical.certification:tutorial
+    stock_reports = text, submission_files
+
+    [test plan]
+    unit = com.canonical.certification::TODO
+    forced = yes
+
+    [test selection]
+    forced = yes
+
+    [environment]
+    TUTO = tutorial
+
+Run Checkbox using this launcher and observe that once the test plan is
+finished running, Checkbox generates a summary on the screen and provides
+the links to the test reports and test archive, but does not ask if the
+result should be uploaded to the Canonical Certification website.
+
+This is thanks to the customization of the ``stock_reports`` field in the
+``[launcher]`` section. If not specified in the launcher, its default value
+is set to ``text, certification, submission_files``.
+
+In Checkbox language, submissions files are the HTML test report as well as
+an archive containing the test results and additional logs that might have
+been produced by the test cases.
+
+Create an executable launcher
+=============================
+
+So far, we have called our launcher using the ``launcher`` option of the
+``checkbox-cli`` tool. It is however possible to turn our launcher into a
+"standalone" executable.
+
+At the top of the launcher file, add this line:
+
+.. code-block:: none
+    :caption: mylauncher
+    :name: launcher-shebang
+    :emphasize-lines: 1
+
+    #!/usr/bin/env checkbox.checkbox-cli
+
+    [launcher]
+    launcher_version = 1
+    app_id = com.canonical.certification:tutorial
+    stock_reports = text, submission_files
+
+    [test plan]
+    unit = com.canonical.certification::TODO
+    forced = yes
+
+    [test selection]
+    forced = yes
+
+    [environment]
+    TUTO = tutorial
+
+Make the launcher executable:
+
+.. code-block:: none
+
+    chmod +x mylauncher
+
+Run it:
+
+.. code-block:: none
+
+    ./mylauncher
+
+Checkbox runs exactly like before! The line we added is called a `shebang
+<https://en.wikipedia.org/wiki/Shebang_(Unix)>`_ and allows us to run
+``checkbox.checkbox-cli`` using the configuration provided.
+
+Wrapping up
+===========
+
+In this section, you've got more familiar with Checkbox launchers. You created
+a launcher that modified the behavior of Checkbox by pre-selecting a test
+plan and executing it, provided environment variables to the test cases and
+outputted only what you needed: a text summary and the submission files.
+
+There are many more options available in the launchers to customize
+Checkbox runs. Please check the :ref:`launchers reference<launcher>` for
+more information.
```

Comment:
or the advanced tutorial

Reply:
I'd point to the reference first. When we get the advanced tutorial, I can also add a link to that (like "in the advanced tutorial, you will see additional ways of customizing...")